src/utils/connect.ts

```diff
@@ -99,6 +99,6 @@ export const decodeBase64UrlMacaroon = (macaroonString: string): string => {
  * @return {String} Encoded macaroon
  */
 export const encodeBase64UrlMacaroon = (macaroonHex: string): string => {
-  const macaroonBase64 = Buffer.from(macaroonHex).toString('base64');
+  const macaroonBase64 = Buffer.from(macaroonHex, 'hex').toString('base64');
```

Comment:
we miss this to properly encode? https://github.com/LN-Zap/node-lndconnect/blob/master/src/encodeMacaroon.js#L16

Reply:
we don't include filepaths in the connect URL. The macaroon is only in base64 format so we don't need that if block.

Reply:
but what when we got the macarron in hex from InitWalletReply?